osu.Game.Modes.Taiko/Beatmaps/TaikoBeatmapConverter.cs

```diff
@@ -41,6 +42,7 @@ public Beatmap<TaikoHitObject> Convert(Beatmap original)
         {
             return new Beatmap<TaikoHitObject>(original)
             {
+                TimingInfo = new LegacyTimingInfo(original.TimingInfo),

```

Comment:
In-fact, maybe not needed to go that far either, and just use `if (original is LegacyBeatmap) ...`